### ISSUE TITLE
linter: vitest/valid-title does not recognize String.raw

### ISSUE THREAD

**Issue by Mister-Hope** [oxc-project/oxc#22268]
### What version of Oxlint are you using?

latest

### What happened?

```ts
it(String.raw`should handle multiple '\'`, () => {
  expect(markdownIt.render(String.raw`~foo\\\\\ bar~`)).toBe("<p><sub>foo\\\\ bar</sub></p>\n");
  expect(markdownIt.render(String.raw`~foo\\\\ bar~`)).toBe("<p>~foo\\\\ bar~</p>\n");
});
```

<img width="842" height="191" alt="Image" src="https://github.com/user-attachments/assets/9bc0ebcd-d633-4675-b253-eb84e383d993" />

I should prefer `String.raw` in my tests as it greatly increase readablilty with `\`
